tools/cjxl.cc

```diff
@@ -578,7 +579,6 @@ jxl::Status CompressArgs::ValidateArgs(const CommandLineParser& cmdline) {
 
   if (progressive) {
     params.qprogressive_mode = true;
-    params.progressive_dc = 1;
```

Comment:
I know we currently don't render DC progressively anyway, but that's a current decoder limitation. Even if we only render something when all the DC is available, it might still be worthwhile to do the DC frame thing though, just because it will have the DC earlier (the DC frame _only_ does DC, while default in-frame DC is mixed with ACmeta). So wouldn't it be better to keep this?

Reply:
I'd prefer it to be explicitly enabled, as it can have surprising effects on quality - `-p` doesn't.

Reply:
OK for me.